src/main/resources/db/changelog/changelog20221025_2350.sql

```diff
@@ -0,0 +1,21 @@
+-- liquibase formatted sql
+
+-- changeset nika.avalishvili:1
+ALTER TABLE benefit
+ADD COLUMN type_id INT,
+ADD COLUMN method_id INT;
+
+-- changeset nika.avalishvili:2
+ALTER TABLE benefit
+    ADD CONSTRAINT fk_benefit_type
+          FOREIGN KEY (type_id)
+            REFERENCES benefit_type(type_id)
+            ON DELETE SET NULL
+
+-- changeset nika.avalishvili:3
+ALTER TABLE benefit
+    ADD CONSTRAINT fk_calculation_method
+          FOREIGN KEY (method_id)
+            REFERENCES calculation_method(method_id)
+            ON DELETE SET NULL
+
```

Comment:
I believe this can be shorten
We can combine creation of the constraint together with creating new column
For example:
https://dba.stackexchange.com/questions/132029/how-to-add-a-column-with-a-foreign-key-constraint-to-a-table-that-already-exists

Reply:
I tried it before, but when I run the app received error notifications "java.lang.IllegalStateException: Failed to load ApplicationContext". I will try again with different ways.

Reply:
Solved!